assets/css/callouts.css

```diff
@@ -6,39 +6,39 @@
  * */
 
 .prose blockquote {
-  @apply p-4 not-italic font-normal bg-blue-light-100 dark:bg-blue-dark-100;
+  @apply bg-blue-light-100 p-4 font-normal not-italic dark:bg-blue-dark-100;
```

Comment:
LOL, I'm trying to read the diff, but some of these lines just seem to be re-ordering the classes; is that part of the fix, or is that your IDE's formatting preference? 🙈 

(if it's just formatting, how much work would it be to split that in a separate commit?)

Reply:
Yeah sorry it was me formatting and being lazy. Let me re-push

Reply:
Thanks!